### ISSUE TITLE
Discusson on Op: fused_batch_norm

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#1672]
@binarman wrote https://github.com/Samsung/ONE/issues/1167#issuecomment-635999572

> **fused_batch_norm**
I did not find this operator in main TF API, but it exists in raw ops: [tf.raw_ops.FusedBatchNorm](https://www.tensorflow.org/api_docs/python/tf/raw_ops/FusedBatchNorm).
Looks like it is same as simple batch norm, but represented in TF graph as one operator.
Unfortunately toco crashed when I tried to convert this operator.
  **I don't see this operator in any of our target networks, maybe postpone it until we find example?**

**Comment by seanshpark** [comment]
> I don't see this operator in any of our target networks, maybe postpone it until we find example?

```py
co = tf.placeholder(tf.float32, shape=[1,4,4,3], name="placeholder")
y,m,v = tf.nn.fused_batch_norm(co, scale=[1.0, 2.0, 3.0], offset=[1.0, 2.0, 3.0],
                               mean=[1.0, 2.0, 3.0], variance=[1.0, 2.0, 3.0],
                               epsilon=0.001, is_training=False, data_format='NHWC',
                               name="FBN_02")
```
This is my test code that I've made sometime ago. There are three outputs.
In `ConvertFusedBatchNormOperator()` of toco, this is imported as Add + Rsqrt + Mul + BatchNorm. 
I think we already support this. But at least we can add an TensorFlow example code like above.
You can send PR of this example.

**Comment by binarman** [comment]
@seanshpark 
> I think we already support this. But at least we can add an TensorFlow example code like above.

Ok, thank you!
Just one last question (a little off-topic): what tensorflow version should we use in python examples?
I am asking because there are mix of normal packages and `compat.v1`

**Comment by seanshpark** [comment]
> what tensorflow version should we use in python examples?

We assume 1.13.x.
Please use `compat.v1`.
Examples without this was made before we used `compat.v1` or wasn't added with `compat.v1`. Currently I don't think seriously with or without `compat.v1`.

**Comment by seanshpark** [comment]
I think this can be closed
